nomenclature/validation.py

```diff
@@ -45,7 +45,11 @@ def validate(dsd, df, dimensions):
             error = True
 
         if invalid_units:
-            lst = [f"{v} - expected: {e}, found: {u}" for v, u, e in invalid_units]
+            lst = [
+                f"'{v}' - expected: {'one of ' if isinstance(e, list) else ''}"
+                f"'{e}', found: '{u}'"
+                for v, u, e in invalid_units
+            ]
             log_error("variable", lst)
```

Comment:
How about also changing the actual text of the error message to
“The following items are reported with the wrong unit:”

Reply:
Updated the error message.